lib/github_changelog_generator/octo_fetcher.rb

```diff
@@ -6,6 +6,7 @@
 require "async/barrier"
 require "async/semaphore"
 require "async/http/faraday"
+require "pry"
```

Comment:
Please remove debugging information from the PR

Reply:
Thank you @carabasdaniel